examples/chip-tool/commands/clusters/BarrierControl/Commands.h

```diff
@@ -0,0 +1,60 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_BARRIERCONTROL_COMMANDS_H__
+#define __CHIPTOOL_BARRIERCONTROL_COMMANDS_H__
+
+#include "../../common/ModelCommand.h"
+
+class MoveToPercent : public ModelCommand
+{
+public:
+    MoveToPercent(const uint16_t clusterId) : ModelCommand("move-to-percent", clusterId) {}
+
+    size_t EncodeCommand(PacketBuffer * buffer, size_t bufferSize, uint16_t endPointId) override
+    {
+        uint8_t percent = 50;
```

Comment:
should the percent be a constructor argument instead?

or if a constant, maybe a public class constant like `static constexpr uint8_t kMovePercent = 50` would be clearer (and then we could document why 50 is good).

Reply:
I have an upcoming PR that will remove all those default values and let the user specify them using chip-tool. I definitely have chosen 50 without good reason except for it between under the max level and above the min level.